### ISSUE TITLE
 Cannot read property 'before' of undefined

### ISSUE THREAD

**Issue by Mehdi-Hp** [jonathantneal/precss#22]
I'm getting this error when I use `@import` in my css file.

```
Plumber found unhandled error:
 TypeError in plugin 'gulp-postcss'
Message:
    Cannot read property 'before' of undefined
```

**Comment by meuwka** [comment]
I have same issue

**Comment by maxisix** [comment]
I have the same !

**Comment by jonathantneal** [comment]
Resolved in 1.0.0 during update to PostCSS 5
